tutorsuperset/plugin.py

```diff
@@ -31,6 +31,8 @@
         ("SUPERSET_OPENEDX_USER_PROFILE_PATH", "/api/user/v1/accounts/{username}"),
         ("SUPERSET_OPENEDX_COURSES_LIST_PATH", "/api/courses/v1/courses/?permissions={permission}&username={username}"),
         ("SUPERSET_OPENEDX_ROLE_NAME", "Open edX"),
+        # Set to 0 to have no row limit.
+        ("SUPERSET_ROW_LIMIT", 100_000_000),
```

Comment:
Is this really the number here? 😳

Reply:
Ah, I should probably use Superset's default here, hey: 
```suggestion
        ("SUPERSET_ROW_LIMIT", 100_000),
```

And yep, the underscores are just for readability :)